### ISSUE TITLE
About Page Content Is Empty

### ISSUE THREAD

**Issue by 0rigin-c0de** [narayan954/dummygram#1251]
### What would you like to share?

![Screenshot 2023-08-14 094107](https://github.com/narayan954/dummygram/assets/106311923/a95b58bd-1434-463d-8db6-cb198547453b)

In dummygram we have an empty.... about page  I think we have to add something in this about the dummygram like how it is made and what it does. And the second one is this when I clicked on about in dark mode the background goes white and we cannot see the about name in the sidebar as you can see in the screenshot above  

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [ ] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
> ### What would you like to share?
> ![Screenshot 2023-08-14 094107](https://user-images.githubusercontent.com/106311923/260356968-a95b58bd-1434-463d-8db6-cb198547453b.png)
> 
> In dummygram we have an empty.... about page I think we have to add something in this about the dummygram like how it is made and what it does. And the second one is this when I clicked on about in dark mode the background goes white and we cannot see the about name in the sidebar as you can see in the screenshot above
> 
> ### Additional information
> _No response_
> 
> ### Record
> * [x]  I agree to follow this project's Code of Conduct
> * [x]  I have checked the existing issues
> * [ ]  I'm a GSSoC'23 contributor
> * [x]  I want to work on this issue

Hi @0rigin-c0de thanks for raising the issue, actually the page you're on is settings page, but we actually do have an about page which you can find in the footer of sidebar in home page of our application. I think perhaps having this about section in sidebar of settings is a bit confusing and illogical. What's your say on this? Could there be anything settings related for about? or should we remove it? Settings page is quite under development. 
And for the second issue, yes sure we can fix that! :)

Regards

**Comment by narayan954** [comment]
or perhaps we can move it down and show about page here as well?

**Comment by 0rigin-c0de** [comment]
yes you are right

**Comment by 0rigin-c0de** [comment]
but second is issue is logical i guess

**Comment by narayan954** [comment]
> but second is issue is logical i guess
> 

yes you're right, you may remove that about section in settings and fix the theme issue :)
